src/passes/ExtractFunction.cpp

```diff
@@ -23,19 +23,25 @@
 
 namespace wasm {
 
-
 struct ExtractFunction : public Pass {
   void run(PassRunner* runner, Module* module) override {
-    auto* leave = getenv("BINARYEN_EXTRACT");
-    if (!leave) {
-      std::cerr << "usage: set BINARYEN_EXTRACT in the env\n";
-      abort();
+    Name name;
+    for (auto& arg : runner->options.arguments) {
```

Comment:
Can we extract this pattern into a more reusable mechanism? E.g. `Name name = runner->getPassOptionPrefixedBy("extract:")`
I predict that as we get more pass args, we'll do that way more.

Reply:
Maybe it'll be a pattern, yeah. But I'm not sure it will be. If it is, we should probably madate it as part of the syntax, so it's no longer arbitrary strings, etc.

Reply:
I was trying to avoid thinking about a more structured design here, because I didn't have a good idea for one, but yeah, I agree it's a little awkward to stop in the middle too, as this PR does.